ulscrape/spiders/uradni_list.py

```diff
@@ -43,13 +42,16 @@ def parse_archive_index_page(self, response):
         archive_pages = range(1, pages_max+1)
 
         for p in archive_pages:
-            yield scrapy.http.FormRequest(url=self.search_url, formdata={'year': str(year), 'page': str(p)}, meta={'year': year, 'page': p})
+            yield FormRequest(url=self.search_url, formdata={'year': str(year), 'page': str(p)}, meta={'year': year, 'page': p})
 
     def parse_archive_page(self, response):
-        yield {
-                'urls': [ self.base_url + url for url in response.css('a[href*=_pdf]::attr(href)').extract() ],
-                'meta': response.meta 
-              }
+        for url in response.css('a[href*=_pdf]::attr(href)').extract():
+            yield UlscrapeItem(
+                    scrapped_at = datetime.now().strftime('%Y-%m-%d %H:%M:%S'),
```

Comment:
I've set `scraped_at` to `utcnow()`, so that you don't have to manually set OR format it,...